### PR TITLE
New package: visual-studio-code-bin-1.25.1

### PR DESCRIPTION
**Pull request by lvitals**

### CODE REVIEW

File: srcpkgs/visual-studio-code-bin/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+license="microsoft"
+repository=nonfree
+restricted=yes
+nostrip=yes
+
+_baseUrl="https://packages.microsoft.com/repos/vscode/pool/main/c/code/"
+_filename="code_${version}-1528912196_amd64.deb"
+_vscodeUrl="${_baseUrl}/${_filename}"
+
+distfiles="${_vscodeUrl}"
```
Comment (cr6git) [inline srcpkgs/visual-studio-code-bin/template]: No, no, no.
Just use:
````
distfiles="https://packages.microsoft.com/repos/vscode/pool/main/c/code/code_${version}-1528912196_amd64.deb"
````

File: srcpkgs/visual-studio-code-bin/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+checksum=7b72995400c679f58e9a7745b751ce066b4692e80d8fdb4ad97b3e79070799b2
+
+do_extract() {
+	mkdir -p ${DESTDIR}
+	ar x ${XBPS_SRCDISTDIR}/visual-studio-code-bin-${version}/${_filename}
+}
+
+do_install() {
+	tar xpf data.tar.xz -C ${DESTDIR}
```
Comment (cr6git) [inline srcpkgs/visual-studio-code-bin/template]: move this part to `do_extract()`

**Comment by travankor** [comment]
new xbps means that xbps can natively build and package vscode (no binary fetching required)

**Comment by cr6git** [comment]
@travankor do you mean we should close this PR?

**Pull request closed without merging**